kotest-assertions/kotest-assertions-core/src/commonMain/kotlin/io/kotest/assertions/nondeterministic/eventually.kt

```diff
@@ -130,30 +133,30 @@ object EventuallyConfigurationDefaults {
 class EventuallyConfigurationBuilder {
 
    /**
-    * The total time that the eventually function can take to complete successfully.
+    * The total time that the [eventually] function can take to complete successfully.
     */
    var duration: Duration = EventuallyConfigurationDefaults.duration
 
    /**
-    * A delay that is applied before the first invocation of the eventually function.
+    * A delay that is applied before the first invocation of the [eventually] function.
     */
    var initialDelay: Duration = EventuallyConfigurationDefaults.initialDelay
 
    /**
-    * The delay between invocations. This delay is overriden by the [intervalFn] if it is not null.
+    * The delay between invocations. This delay is override by the [intervalFn] if it is not `null`.
     */
    var interval: Duration = EventuallyConfigurationDefaults.interval
 
    /**
-    * A function that is invoked to calculate the next interval. This if this null, then the
+    * A function that is invoked to calculate the next interval. This if this `null`, then the
     * value of [interval] is used.
     *
-    * This function can be used to implement [fibonacci] or [exponential] backoffs.
+    * This function can be used to implement [fibonacci] or [exponential] backoff.
```

Comment:
```suggestion
    * This function can be used to implement a [fibonacci] or an [exponential] backoff.
```

Reply:
I much prefer the original one, omitting the articles reads more naturally and is more direct.

Reply:
I don't know why as I usually can't explain language rules anyway, but the first variant seemed to long for a plural form "backoffs". But I don't mind.